cgmanifests/generated/cgmanifest.json

```diff
@@ -92,7 +92,7 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "0ab19024f08c6673a713e454ef8bd95e174c807f",
+          "commitHash": "c220895fd1163c01f0a8b44229fb9e4fe0ae0958",
```

Comment:
Run "git submodule update --init --recursive" before generating this file.

Reply:
Thanks for the heads up. This change has been restored